SeaPublicWebsite/Views/EnergyEfficiency/YourRecommendations.cshtml

```diff
@@ -42,9 +42,13 @@
             <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
             <strong class="govuk-warning-text__text">
                 <span class="govuk-warning-text__assistive">Note</span>
-                The costs and savings we provide are estimates and may not be the same in practice. 
-                The savings displayed are based on the October 2022 energy price cap. 
-                As energy prices change, so will the estimates of savings.
+                The costs and savings we provide are estimates
+                and may not be the same in practice. The savings
+                displayed are based on the January 2024 energy
+                price cap for gas and electricity bills, and on the
+                RdSAP energy prices for all other heating fuel
+                types. As energy prices change, so will the
+                estimates of savings.
```

Comment:
I'd prefer keeping the style of the original text here and having one sentence per line.

It makes the breaks more readable and the changes in the commit more obvious.

Typically, I break paragraphs in code after each full stop. If that makes a line seem too long, then look for an appropriate place to break the sentence, like a comma or an "and".

In this case, that would be:

```
The costs and savings we provide are estimates and may not be the same in practice.
The savings displayed are based on the January 2024 energy price cap for gas and electricity bills,
and on the RdSAP energy prices for all other heating fuel types.
As energy prices change, so will the estimates of savings.
```